LICENSE

```diff
@@ -2,14 +2,11 @@ The MIT License (MIT)
 
 Copyright (c) 2016 Yuki Hattori (yukihattori1116@gmail.com)
 
-This software also uses portions of the KaTeX and underscore.js projects, which are
```

Comment:
Sorry, busy week took me a while to get around to reading this. Thanks for the explanation, yes that makes sense!